lib/sql.js

```diff
@@ -1185,8 +1185,15 @@ SQLConnector.prototype.buildOrderBy = function(model, order) {
     return '';
   }
   var self = this;
-  if (typeof order === 'string') {
-    order = [order];
+  var supportedConnectors = ['mysql', 'postgresql', 'oracle', 'mssql'];
+  var usedConnector = this.dataSource.connector.name;
+  if (order === '${random}') {
```

Comment:
My initial proposal is to allow `${var}` in the filter object in general, not only for `random`. We should parse the filter object and resolve vars (from context and connector) before invoking the driver command.